### ISSUE TITLE
App Footer Obscured by URL Bar

### ISSUE THREAD

**Issue by Sdaswani** [mozilla-mobile/fenix#552]
## Steps to reproduce
See this video: https://photos.app.goo.gl/YLgn66HoYyvDc2VK7

### Expected behavior
When Fenix scrolling is at rest, it would be optimal if the web app's footer control bar re-appears.

### Actual behavior
Web app's footer control bar is obscured by URL bar.

### Device information

* Android device: Galaxy S8
* Fenix version: Today's nightly

This not a _huge deal_, but it's nice to have working so users choose their social media on Fenix instead of in app.

**Comment by TitanNano** [comment]
See my comment here: https://github.com/mozilla-mobile/fenix/issues/173#issuecomment-460979452
I think it would be enough if the browser toolbar could be easily dismissed.

**Comment by topotropic** [comment]
Expected behavior here is that the web app/website bottom bar renders above the address bar

Related:
 - #830 
 - https://github.com/mozilla-mobile/reference-browser/issues/464

**Comment by vesta0** [comment]
Related GV bugs: 
[1536222](https://bugzilla.mozilla.org/show_bug.cgi?id=1536222)
[1516048](https://bugzilla.mozilla.org/show_bug.cgi?id=1516048)

**Comment by vesta0** [comment]
This is considered a release blocker due to impact on usability. Other notable examples of sites impacted:

- instagram
- pinterest
- google maps
- digg.com
- Reddit
- Quora bottom nav bar
- flights.google.com: the airport search mode's Cancel/Apply buttons

**Comment by lime124** [comment]
i'm not clear who's court the ball is in. from my standpoint it appears that it's on eng to fix as per nicole's comment "the content should render above the url bar". To me that includes that handle piece of UI which is the primary thing that is causing problems...?

**Comment by boek** [comment]
@lime124 The real bug is we don't want the address bar to cover the webpage. There are some open GV issues to address this, once fixes land we will make sure it's also fixed in Fenix :+1:

**Comment by lime124** [comment]
Got it thanks boek - so the issue is blocked on those GV bugs getting fixed. Thanks!

**Comment by vesta0** [comment]
@lime124 the GeckoView team requires UX feedback on how this should behave before they can fix it. See email from @cpeterso

**Comment by lime124** [comment]
@topotropic just respond to @cpeterso via email with this. I believe this should resolve the issue. 

![unnamed](https://user-images.githubusercontent.com/287322/55773130-22d26300-5a44-11e9-8607-f4afa0b9a245.png)

**Comment by vesta0** [comment]
Will be addressed: A-C bug: https://github.com/mozilla-mobile/android-components/issues/2675

**Comment by 1jj** [comment]
I was about filing a related issue with  http://bela-lugosi.de (URL bar should not cover content), using build from yesterday.

Are you aware of   **"Intent to Ship: Top Controls don't resize initial containing block and vh units"** ?
https://groups.google.com/a/chromium.org/forum/#!msg/blink-dev/BK0oHURgmJ4/PZqveT2LBAAJ

In Chrome 56 Blink followed iOS and endet up with implementing something like this:

`height:100% != height:100vh != window.innerHeight !=documentElement.clientHeight != position:absolute;bottom:0 `

It  depends on presence of URL bar. https://www.chromestatus.com/feature/6241601772847104

They filed a bug against Firefox  https://bugzilla.mozilla.org/show_bug.cgi?id=1314686 ,
confirmed by kats@mozilla.com

**Comment by reinhart1010** [comment]
Referencing webcompat/web-bugs#30094 from @webcompat

**Comment by cadeyrn** [comment]
@pocmo Since you implemented the changes for this issue please note #2574: While the content is no longer hidden you still can't interact with the previous hidden content until the toolbar is hidden.

**Comment by softvision-miralobontiu** [comment]
I can confirm that the app's footer is no longer obscured, but the footer is not entirely visible.
Tested on the latest build 1.0.1920 (#11361806), with Sony Xperia Z5 Premium (Android 7.1.1), and HTC 10 (Android 8).
Also, #2574 is reproducible on Instagram too.

![Screenshot_1](https://user-images.githubusercontent.com/40694158/57930061-a857f700-78bd-11e9-95be-3774d0ab1f46.png)

**Comment by Amejia481** [comment]
@softvision-miralobontiu this issue will be covered in https://github.com/mozilla-mobile/fenix/issues/2563

**Comment by TitanNano** [comment]
It doesn't work on Google Maps though. 
![Screenshot_20190517-154024](https://user-images.githubusercontent.com/4500858/57932124-75603400-78ba-11e9-96a2-3921b03644e3.png)
![Screenshot_20190517-154020](https://user-images.githubusercontent.com/4500858/57932125-75f8ca80-78ba-11e9-91fa-7206cbd13415.png)

**Comment by softvision-miralobontiu** [comment]
Yes, unfortunately, it is still reproducible on maps.google.com. Tested on the latest build 1.0.1920 (#11361806), with Sony Xperia Z5 Premium (Android 7.1.1), and HTC 10 (Android 8).
@ekager would you mind taking a look?

**Comment by ekager** [comment]
Google Maps will be dealt with by https://github.com/mozilla-mobile/fenix/issues/2154

**Comment by ekager** [comment]
I think we can close this issue for #2563 and #2154
